onnxruntime/core/framework/op_kernel_context_internal.h

```diff
@@ -53,6 +53,10 @@ class OpKernelContextInternal : public OpKernelContext {
     return OpKernelContext::GetOutputMLValue(index);
   }
 
+  void SetOutputMLValue(int index, const OrtValue& ort_value) {
+    ORT_ENFORCE(OpKernelContext::SetOutputMLValue(index, ort_value).IsOK());
```

Comment:
Is it possible to avoid ORT_ENFORCE? Unfortunately, it looks like we don't necessarily have a well defined policy around this.

Reply:
The underlaying function can only fail if the index is out of range. 
Do you think it's better to simply return the status to the caller of this function ?

Reply:
yeah, just being consistent with the above OpKernelContext::SetOutputMLValue.

Reply:
returning status code directly to the caller